gapic-generator-cloud/templates/cloud/gem/readme.erb

```diff
@@ -2,10 +2,18 @@
 
 <%= gem.summary %>
 
-<%= gem.description %>
+<%= gem.readme_description %>
 
 <%= gem.homepage %>
 
+<%- if gem.wrapper_name -%>
+This gem is a _versioned_ client. It provides basic client classes for a
+specific version of the <%= gem.title %> API. Most users should consider the
+[<%= gem.wrapper_name %>](https://rubygems.org/gems/<%= gem.wrapper_name %>)
+gem, a convenience wrapper that may also include higher-level interface classes.
```

Comment:
"convenience wrapper" -> "main client"

Reply:
I would prefer to see the word 'wrapper' because there is no client class in there for most of them. 
Same for the text below; in any case those should be unified.

Reply:
I'm not opposed to "wrapper" instead of "main". My suggestion above is purely for consistency.

Reply:
I moved away from the "wrapper" terminology. Not all of these are wrappers, and it's kind of not the message we want to send. We want to say "**this** is **the** client," rather than "this is just a wrapper around the client".